dependencies.props

```diff
@@ -76,6 +79,14 @@
       <BuildInfoPath>$(BaseDotNetBuildInfo)projectn-tfs/$(DependencyBranch)</BuildInfoPath>
       <CurrentRef>$(ProjectNTfsCurrentRef)</CurrentRef>
     </RemoteDependencyBuildInfo>
+    <RemoteDependencyBuildInfo Condition="'$(DisableProjectNTfsAutoUpgrade)' != 'true'" Include="ProjectNTfs-TestILC">
```

Comment:
if updating the ProjectN Targeting pack is conditioned, then we won't want to update this as well. Now I'm not sure why would we not want to update ProjectN itself, but it was already conditioned so all I did was too keep it consistent and have the same condition on it.